### ISSUE TITLE
Unable to find Arial.ttf

### ISSUE THREAD

**Issue by tinypirate08** [pbnjay/lollipops#13]
Hi I tried this handy littke thing on both Mac with Yosemite 10.10.5 and on my CentOS 6.6 box.
I downloaded the binaries and started with a simple command for a single gene.

I am getting the same error on both.

-bash-4.1$ ./lollipops TAF1 S1579G
ERROR: Unable to find Arial.ttf - Required for accurate font sizing.
       Please use -f=/path/to/arial.ttf or the TTF font of your choice.

Any ideas what may be going on?
Thanks

**Comment by pbnjay** [comment]
If somehow you don't have the Font "Arial" installed on your system, then provide the path to something else for the image generation to use for labels. Arial is part of a typical install for Yosemite so I don't know why you wouldn't have it. On your Mac, there should be a bunch of fonts you can list with `ls /Library/Fonts/*.ttf` but on CentOS it's probably under `/usr/share/fonts/truetype/` I can't really help much with this issue other than pointing out the usage described in the error message.

For example, if you want to use another typical Mac OS X font called Andale Mono:
```
$ ./lollipops -f="/Library/Fonts/Andale Mono.ttf" -o test.png TAF1 S1579G
HGNC Symbol:  TAF1
Uniprot returned 46 hits for your gene symbol 'TAF1':
Entry	Entry name	Status	Gene names	Organism
P04637	P53_HUMAN	reviewed	TP53 P53	Homo sapiens (Human)
P21675	TAF1_HUMAN	reviewed	TAF1 BA2R CCG1 CCGS TAF2A	Homo sapiens (Human)
Q01105	SET_HUMAN	reviewed	SET	Homo sapiens (Human)
P06400	RB_HUMAN	reviewed	RB1	Homo sapiens (Human)
Q03164	KMT2A_HUMAN	reviewed	KMT2A ALL1 CXXC7 HRX HTRX MLL MLL1 TRX1	Homo sapiens (Human)
P51610	HCFC1_HUMAN	reviewed	HCFC1 HCF1 HFC1	Homo sapiens (Human)
Q9Y230	RUVB2_HUMAN	reviewed	RUVBL2 INO80J TIP48 TIP49B CGI-46	Homo sapiens (Human)
Q15545	TAF7_HUMAN	reviewed	TAF7 TAF2F TAFII55	Homo sapiens (Human)
Q9Y265	RUVB1_HUMAN	reviewed	RUVBL1 INO80H NMP238 TIP49 TIP49A	Homo sapiens (Human)
P52655	TF2AA_HUMAN	reviewed	GTF2A1 TF2A1	Homo sapiens (Human)
Q99496	RING2_HUMAN	reviewed	RNF2 BAP1 DING HIPI3 RING1B	Homo sapiens (Human)
Q9HCK8	CHD8_HUMAN	reviewed	CHD8 HELSNF1 KIAA1564	Homo sapiens (Human)
Q16594	TAF9_HUMAN	reviewed	TAF9 TAF2G TAFII31	Homo sapiens (Human)
P20226	TBP_HUMAN	reviewed	TBP GTF2D1 TF2D TFIID	Homo sapiens (Human)
Q8IZX4	TAF1L_HUMAN	reviewed	TAF1L	Homo sapiens (Human)
Q9H7Z6	KAT8_HUMAN	reviewed	KAT8 MOF MYST1 PP7073	Homo sapiens (Human)
Q53T94	TAF1B_HUMAN	reviewed	TAF1B	Homo sapiens (Human)
P61244	MAX_HUMAN	reviewed	MAX BHLHD4	Homo sapiens (Human)
Q9Y294	ASF1A_HUMAN	reviewed	ASF1A CGI-98 HSPC146	Homo sapiens (Human)
P35269	T2FA_HUMAN	reviewed	GTF2F1 RAP74	Homo sapiens (Human)
P61964	WDR5_HUMAN	reviewed	WDR5 BIG3	Homo sapiens (Human)
Q8WWY3	PRP31_HUMAN	reviewed	PRPF31 PRP31	Homo sapiens (Human)
P49848	TAF6_HUMAN	reviewed	TAF6 TAF2E TAFII70	Homo sapiens (Human)
Q96EZ8	MCRS1_HUMAN	reviewed	MCRS1 INO80Q MSP58	Homo sapiens (Human)
Q9UBL3	ASH2L_HUMAN	reviewed	ASH2L ASH2L1	Homo sapiens (Human)
Q7Z3B3	KANL1_HUMAN	reviewed	KANSL1 CENP-36 KIAA1267 MSL1V1 NSL1	Homo sapiens (Human)
Q96IU4	ABHEB_HUMAN	reviewed	ABHD14B CIB	Homo sapiens (Human)
Q9NVP2	ASF1B_HUMAN	reviewed	ASF1B	Homo sapiens (Human)
O75461	E2F6_HUMAN	reviewed	E2F6	Homo sapiens (Human)
Q9Y262	EIF3L_HUMAN	reviewed	EIF3L EIF3EIP EIF3S6IP HSPC021 HSPC025 MSTP005	Homo sapiens (Human)
Q6PI98	IN80C_HUMAN	reviewed	INO80C C18orf37	Homo sapiens (Human)
Q9Y4W2	LAS1L_HUMAN	reviewed	LAS1L MSTP060	Homo sapiens (Human)
Q8IWI9	MGAP_HUMAN	reviewed	MGA KIAA0518 MAD5	Homo sapiens (Human)
Q8IZL8	PELP1_HUMAN	reviewed	PELP1 HMX3 MNAR	Homo sapiens (Human)
Q9BVI0	PHF20_HUMAN	reviewed	PHF20 C20orf104 GLEA2 HCA58 NZF TZP	Homo sapiens (Human)
Q15291	RBBP5_HUMAN	reviewed	RBBP5 RBQ3	Homo sapiens (Human)
Q3B726	RPA43_HUMAN	reviewed	TWISTNB	Homo sapiens (Human)
Q9NYV6	RRN3_HUMAN	reviewed	RRN3 TIFIA	Homo sapiens (Human)
Q9H4L4	SENP3_HUMAN	reviewed	SENP3 SSP3 SUSP3	Homo sapiens (Human)
Q15542	TAF5_HUMAN	reviewed	TAF5 TAF2D	Homo sapiens (Human)
Q7Z7C8	TAF8_HUMAN	reviewed	TAF8 TAFII43 TBN	Homo sapiens (Human)
Q5H9L4	TAF7L_HUMAN	reviewed	TAF7L TAF2Q	Homo sapiens (Human)
O00268	TAF4_HUMAN	reviewed	TAF4 TAF2C TAF2C1 TAF4A TAFII130 TAFII135	Homo sapiens (Human)
Q9NXF1	TEX10_HUMAN	reviewed	TEX10 L18 Nbla10363	Homo sapiens (Human)

Selected 'Q53T94' as the best match. Use -U XXX to use another ID.

Uniprot/SwissProt Accession:  Q53T94
Drawing diagram to test.png

```

You probably don't want the default selection there, use the appropriate Uniprot ID (with -U XXXX):

```
$ ./lollipops -f="/Library/Fonts/Andale Mono.ttf" -o test.png -U P21675 S1579G   
Pfam Symbol:  TAF1_HUMAN
Drawing diagram to test.png

```
![test](https://cloud.githubusercontent.com/assets/161552/12730632/d0d03506-c8fa-11e5-9707-5e8cdaa81cf2.png)

**Comment by tinypirate08** [comment]
Love it! It works now. Thanks for the advice. The plot is amazing!

**Comment by tinypirate08** [comment]
Hi Jeremy,
One quick question. Is there a way to control the domain name labels within the colored domain bars?

What if I did not want to include the text?
Looks like if I adjust image size, the text gets truncated within domains. See green and blue bars before and after.

Is there a way to adjust text size, or not to include?

I used command for TAF1.png
./lollipops -f=/Library/Fonts/Courier\ New.ttf -labels -hide-motifs -o TAF1.png -U P21675 S1579G

I used the follwing for TAF1_2.png
/lollipops -f=/Library/Fonts/Courier\ New.ttf -labels -hide-motifs  -w 6000-dpi 300 -o TAF1_2.png -U P21675 S1579G

I tried multiple size combinations but, there is always one domain name that is truncated.

Thanks for your input.
Szabi



[cid:0B472A7B-C1D8-4ECC-9401-98B18832C28E@ad.tgen.org]

[cid:6D5F3841-C363-4D15-96B3-FEEF26D79ECC@ad.tgen.org]


Szabolcs Szelinger, Ph.D
Staff Scientist
Neurogenomics Division, The Translational Genomics Research Institute (TGen)
445 N. 5th Street, Phoenix AZ 85044
ph:602-343-8723
sszelinger@tgen.org<mailto:sszelinger@tgen.org>



________________________________
________________________________

This electronic message is intended to be for the use only of the named recipient, and may contain information that is confidential or privileged, including patient health information. If you are not the intended recipient, you are hereby notified that any disclosure, copying, distribution or use of the contents of this message is strictly prohibited. If you have received this message in error or are not the named recipient, please notify us immediately by contacting the sender at the electronic mail address noted above, and delete and destroy all copies of this message. Thank you.

On Feb 1, 2016, at 14:03, Jeremy Jay <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #13<https://github.com/pbnjay/lollipops/issues/13>.
